lesson_2/Section_2.2.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<p>we eat we eat</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>